Group 1 and 2 Tutorials for Get Started with SAP Analytics Cloud

andrewliu-SAP opened the pull request:
Reviewed by Developer Advocate Witalij Rudnicki & Joshua Margo

MichaelCzcz commented on tutorials/sac-analytics-designer-create-1-create/sac-analytics-designer-create-1-create.md:
```diff
@@ -0,0 +1,123 @@
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](XXXXX)]
```
Why XXXXX?

andrewliu-SAP replied:
This push was a mistake and is for another Group of Tutorials I'm implementing.

MichaelCzcz commented on tutorials/sac-intro-2-trial/sac-intro-2-trial.md:
```diff
@@ -0,0 +1,71 @@
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](SAP Analytics Cloud Trial Overview)]
```
Please move this to the description section, since it contains only further information, which is not part of the tutorial.

andrewliu-SAP replied:
Moved. Thanks

MichaelCzcz commented on tutorials/sac-intro-2-trial/sac-intro-2-trial.md:
```diff
@@ -0,0 +1,71 @@
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Register for an SAP.com Account)]
```
Please move this to the prerequisites section (remove the image).

andrewliu-SAP replied:
Our tutorials are tailored to to those looking to Get Started with SAC and so having an SAP account would not be a prerequisite. Instead, we want to show them how to get an account, trial, etc.

MichaelCzcz commented on tutorials/sac-intro-3-tour/sac-intro-3-tour.md:
```diff
@@ -0,0 +1,119 @@
+
+The sidebar navigation is where you access the main applications and features of SAP Analytics Cloud including Stories, Analytic Applications, Modeling, Data Analyzer, Digital Boardroom, and more. It is located on the left side of the page and can be collapsed and expanded. The navigation bar will always be present, no matter what application you are working in, making it quick and easy for you t ...
+
+> The options that you see in your side navigation is dependent on your standard application role and permission access. Administrators have full access to all the content creation and administration tools, whereas analytic consumers see a simpler menu with a reduced set of items.
```
Please use correct formatting for notes.

MichaelCzcz commented on tutorials/sac-story-explore-1-open/sac-story-explore-1-open.md:
```diff
@@ -0,0 +1,128 @@
+[DONE]
+[ACCORDION-END]
+
+<!--
```
Please remove tutorials parts that are not used.

andrewliu-SAP replied:
Done. Thanks

MichaelCzcz commented on tutorials/sac-story-explore-2-get-familiar/sac-story-explore-2-get-familiar.md:
```diff
@@ -0,0 +1,93 @@
+|  6              | **Controls**            | View all filters applied to the page and widgets.
+|  7              | **Edit/View**           | Toggle between Edit and View modes. Edit mode enables a Design panel and a larger toolbar filled with features and capabilities that you can use to build stunning and interactive dashboards and reports.  Note that Edit mode is available only if you have rights to edit the story.
+
+<!-- 1. **File** - Go here to export, share, schedule, and publish your story to the catalog.
```
Please remove sections that are not used.

andrewliu-SAP replied:
Done. Thanks

MichaelCzcz reviewed (changes requested):
Thank you this pull request. There are issues regarding style and formatting that you need to fix. I have only commented on some of them, because there were just too many issues.
Please do not use the "royal we" unless you are referring to SAP. Use "you" when addressing the user.
Please don't use phrases like "click on the button" or "go to". Use words like "choose", "select", "navigate to/back" instead.
Regarding the formatting issues, please read the [style guide](https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true) and make all necessary changes. This includes file names. 
There is also one image that need to be cropped, because you can see the checkered background of the graphics program.

andrewliu-SAP commented:
Thanks for your review and comments Michael. I have updated the comments you made on Github, and changed all the "we" to "you". As for the "click"s, I only use that when directing users to learn more from the help guide - "Click here to learn more about...". Everything that is directing the user to click on something in the SAC tenant I refer to "select", "choose", etc.

For the image that needs to be cropped, I did not find any image with a checkered background in there. If you're referring to a transparent background, it does not show on the QA site as checkered. Please let me know which image you're talking about.

Thanks again for your comments! They did make the tutorials better so thank you. I have committed the changes and pushed them to PR (https://github.com/sap-tutorials/Tutorials/compare/master...andrewliu-SAP:master). Would really appreciate your help in getting these through so that they can be put into Missions and Groups by Joshua.

Best,
Andrew

From: Michael Czadankiewicz ***@***.***>
Sent: March 17, 2022 12:49 AM
To: sap-tutorials/Tutorials ***@***.***>
Cc: Liu, Andrew ***@***.***>; Author ***@***.***>
Subject: Re: [sap-tutorials/Tutorials] Group 1 and 2 Tutorials for Get Started with SAP Analytics Cloud (PR #18217)


@MichaelCzcz requested changes on this pull request.

Thank you this pull request. There are issues regarding style and formatting that you need to fix. I have only commented on some of them, because there were just too many issues.
Please do not use the "royal we" unless you are referring to SAP. Use "you" when addressing the user.
Please don't use phrases like "click on the button" or "go to". Use words like "choose", "select", "navigate to/back" instead.
Regarding the formatting issues, please read the style guide<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fjam4.sapjam.com%2Fwiki%2Fshow%2FMozDWaPczznL8NKeGEk0y9%3F_lightbox%3Dtrue&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=q%2FVsu9LzaOpHUXesjEn2cnsT6dwYCgDksFBEKJHsVGU%3D&reserved=0> and make all necessary changes. This includes file names.
There is also one image that need to be cropped, because you can see the checkered background of the graphics program.

________________________________

In tutorials/sac-analytics-designer-create-1-create/sac-analytics-designer-create-1-create.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23discussion_r828821326&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=12mcEu4kVfiohDDbgpHfsP9IdlLmdsaq96GAQBNiAAk%3D&reserved=0>:

> +```

+

+![2.1.10](2.1.10.jpg)

+

+When you run the application, you only want to see the chart. To hide the table at startup time, uncheck the option "Show this item at view time".

+

+![2.1.11](2.1.11.jpg)

+

+Run the application and click the button multiple times. The table or the chart are displayed alternatively.

+

+![2.1.12](2.1.12.gif)

+

+[DONE]

+[ACCORDION-END]

+

+[ACCORDION-BEGIN [Step 4: ](XXXXX)]

Why XXXXX?

________________________________

In tutorials/sac-intro-2-trial/sac-intro-2-trial.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23discussion_r828824408&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=jxb%2BjTKUB1J2qQWodiZ973334r7klN2Q%2BmDs5pIJNOk%3D&reserved=0>:

> +

+<!-- ## Prerequisites

+ - Prerequisite 1

+ - Prerequisite 2 -->

+

+## Details

+### You will learn

+  - What is included in the trial

+  - How to register an SAP.com account

+  - How to register for an SAP Analytics Cloud Trial account

+

+<!-- Add additional information: Background information, longer prerequisites -->

+

+---

+

+[ACCORDION-BEGIN [Step 1: ](SAP Analytics Cloud Trial Overview)]

Please move this to the description section, since it contains only further information, which is not part of the tutorial.

________________________________

In tutorials/sac-intro-2-trial/sac-intro-2-trial.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23discussion_r828825225&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=fsNL%2FjlSMxMq4k9j1wwoR85vMF%2Fnu1JSjlxDzbDq%2BBg%3D&reserved=0>:

> +  - How to register for an SAP Analytics Cloud Trial account

+

+<!-- Add additional information: Background information, longer prerequisites -->

+

+---

+

+[ACCORDION-BEGIN [Step 1: ](SAP Analytics Cloud Trial Overview)]

+

+The free trial of SAP Analytics Cloud gives you a single-user license for SAP Analytics Cloud, with access to business intelligence, enterprise planning, and augmented analytics features. You can connect your data sources, model your data, create stories, try augmented analytics, and do predictive planning. Click [here](https://community.sap.com/topics/cloud-analytics/faq#free-trial) to learn more.

+

+The trial length is 30 days but can be extended for an additional 60 days after the first few weeks of activation.

+

+[DONE]

+[ACCORDION-END]

+

+[ACCORDION-BEGIN [Step 2: ](Register for an SAP.com Account)]

Please move this to the prerequisites section (remove the image).

________________________________

In tutorials/sac-intro-3-tour/sac-intro-3-tour.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23discussion_r828827357&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=zjnmg8CK8fdRsAyTmYwgxhbjbztATLLG0Fet4hDa7Ts%3D&reserved=0>:

> +

+You can toggle between four tabs on the Home Screen:

+

+- **Today**: This is the default tab and shows your home screen

+- **Catalog**: Single access point for content published for users

+- **Favorites**: Lists the resources you have "starred" as your favorite

+- **Shared With Me**: Lists all the resources shared with you by other users

+

+[DONE]

+[ACCORDION-END]

+

+[ACCORDION-BEGIN [Step 2: ](Sidebar Navigation)]

+

+The sidebar navigation is where you access the main applications and features of SAP Analytics Cloud including Stories, Analytic Applications, Modeling, Data Analyzer, Digital Boardroom, and more. It is located on the left side of the page and can be collapsed and expanded. The navigation bar will always be present, no matter what application you are working in, making it quick and easy for you to switch between different applications.

+

+> The options that you see in your side navigation is dependent on your standard application role and permission access. Administrators have full access to all the content creation and administration tools, whereas analytic consumers see a simpler menu with a reduced set of items.

Please use correct formatting for notes.

________________________________

In tutorials/sac-story-explore-1-open/sac-story-explore-1-open.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23discussion_r828828446&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=gD%2FSYtkaxqmx94DbzEP4mRSPGOwpH%2BKA65FDRp6lzXY%3D&reserved=0>:

> +|1                |  **Numeric Point and Bullet Charts** | Visualizes the status of Key Performance Indicators (KPI).  You can immediately see the current KPI value compared to your business's performance thresholds.

+|2                |  **Time Series Chart**               | Displays trends in any kind of numeric data over time.  Is your business trending positively?

+|3                | **Input Control Filters**            | Allows you to focus in on data segments.  Slide the Date Range filter to select your dates for analysis and the filter checkboxes helps you be precise in your analysis.

+|4                |  **Bar Charts**                      | Visualizes values across a Series of data.  As an example, the first chart on the left shows Gross Margin % by the Product and Location dimensions. They're easy to understand and interact with by drilling down, sorting, or analyzing with Smart Insights.  We'll use these together in the later tutorials.

+

+<!--

+1. Numeric Point Charts

+2. Bullet Chart

+3. Input Control Filters

+4. Time Series Chart

+5. Bar Charts -->

+

+[DONE]

+[ACCORDION-END]

+

+<!--

Please remove tutorials parts that are not used.

________________________________

In tutorials/sac-story-explore-2-get-familiar/sac-story-explore-2-get-familiar.md<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23discussion_r828828987&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=F3%2FRU9nJDLaFCPOboq0YK%2FhGR%2F4tWBiHHxTOWQ5dAMI%3D&reserved=0>:

> +Let's first look at the toolbar. In the toolbar, you can save a copy, share with other users, add filters, navigate pages, toggle between View and Edit modes, and more.

+

+![Toolbar](Toolbar resized.png)

+

+

+|  Number         | Name                    | Description

+|  :------------- | :-------------          | :-------------

+|  1              | **File**                | Go here to export, share, schedule, and publish your story to the catalog.

+|  2              | **Edit**                | Reset your story to its original view state, refresh your story data, copy, and paste widgets.

+|  3              | **Tools**               | Edit Prompts for your data models, bookmark the current state of your story, run formulas on table cells, and  create Story Filters that filter data story-wide.

+|  4              | **Display**             | Toggle Comment Mode and Tab Display (page navigation option) on or off. We'll use these in Step 2.

+|  5              | **Quick Access Icons**  | Apply story filters, refresh data, and add bookmarks with just one click.

+|  6              | **Controls**            | View all filters applied to the page and widgets.

+|  7              | **Edit/View**           | Toggle between Edit and View modes. Edit mode enables a Design panel and a larger toolbar filled with features and capabilities that you can use to build stunning and interactive dashboards and reports.  Note that Edit mode is available only if you have rights to edit the story.

+

+<!-- 1. **File** - Go here to export, share, schedule, and publish your story to the catalog.

Please remove sections that are not used.

-
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23pullrequestreview-912729575&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=UaxlGhP5mtLYyciHXhhSjw1eaxQD%2BvPgHqndHG5rjHU%3D&reserved=0>, or unsubscribe<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAVXO4EQC4NUHURV6LZNXOB3VALPYRANCNFSM5Q4HSEWQ&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719338740%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Ou1w6Zj25re0JOD95fcMpIjLJKpBjc48HDaJy6zQBRs%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719494959%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=lQO0abpokqY2auXearufNc2895A9s0jrb7gs09o6rBM%3D&reserved=0> or Android<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Candrew.liu02%40sap.com%7Cab4c1dc9451242adf39708da07eaaabb%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831001719494959%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=hAYD3G2lGic7SBwxzMtwT0teGKxJPvWoeJOsJoslU6c%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>

andrewliu-SAP commented:
Thanks Joshua for the merge! Here are the info on Groups and Missions again.

Mission
Info
Value
Mission title
Get Started with SAP Analytics Cloud
Mission description
Learn how to get started with SAP Analytics Cloud. Along with instructions on how to get a trial account, you will get a tour of the basics of SAP Analytics Cloud, learn how to create and interact with stories, and utilize the power of augmented analytics with SAP Analytics Cloud flagship features like Smart Insights and Search to Insight.
Experience
Beginner
Primary Tag
SAP Analytics Cloud
Secondary Tags
Cloud
Time to Complete
2 hours 15 mins
Group 1
Info
Value
Group title
Get to know SAP Analytics Cloud
Group description
Get an introduction to the User Experience and capabilities of SAP Analytics Cloud
Experience
Beginner
Primary Tag
SAP Analytics Cloud
Secondary Tags
Cloud
Time to Complete
25 mins
Group 2
Info
Value
Group title
Explore the SAP Analytics Cloud Sample Story
Group description
Get an introduction to interacting with story elements and employ the power of Smart Features offered with SAP Analytics Cloud.
Experience
Beginner
Primary Tag
SAP Analytics Cloud
Secondary Tags
Cloud
Time to Complete
45 mins
Group 3
Info
Value
Group title
Create your First SAP Analytics Cloud Story
Group description
Create your first story through importing a dataset and visualizing your data with charts, tables, and geo maps.
Experience
Beginner
Primary Tag
SAP Analytics Cloud
Secondary Tags
Cloud
Time to Complete
65 mins

Best,
Andrew

From: Joshua Margo ***@***.***>
Sent: March 17, 2022 1:43 PM
To: sap-tutorials/Tutorials ***@***.***>
Cc: Liu, Andrew ***@***.***>; Author ***@***.***>
Subject: Re: [sap-tutorials/Tutorials] Group 1 and 2 Tutorials for Get Started with SAP Analytics Cloud (PR #18217)


Merged #18217<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217&data=04%7C01%7Candrew.liu02%40sap.com%7C1b2da479aabd4b47cc7008da0856c879%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831466128823995%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Dj8VhU3t%2FQA8jOrGUNLoSZtInPmUHRJ1AmSJw6jcBJs%3D&reserved=0> into master.

-
Reply to this email directly, view it on GitHub<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsap-tutorials%2FTutorials%2Fpull%2F18217%23event-6262016360&data=04%7C01%7Candrew.liu02%40sap.com%7C1b2da479aabd4b47cc7008da0856c879%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831466128823995%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=rhsWLrrhHtfkMS1f5rQcNasGVfBsD9pKV77wWVgarU0%3D&reserved=0>, or unsubscribe<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAVXO4EQIGPQNRSHSCAVT6TDVAOKOZANCNFSM5Q4HSEWQ&data=04%7C01%7Candrew.liu02%40sap.com%7C1b2da479aabd4b47cc7008da0856c879%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831466128823995%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=0k3LtXD1rX1%2B8VVaDlQ0yZ9roJCid8GIQgX6xpAfBCk%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Candrew.liu02%40sap.com%7C1b2da479aabd4b47cc7008da0856c879%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831466128823995%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=mROsxh32Ohs7%2FU5d%2B%2Fascew9rZm2qMjEkOR38LuLwuo%3D&reserved=0> or Android<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Candrew.liu02%40sap.com%7C1b2da479aabd4b47cc7008da0856c879%7C42f7676cf455423c82f6dc2d99791af7%7C0%7C0%7C637831466128823995%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=w1gOUx3r6vohNynaXw9JtgXkXrt4LUtq3GrZrTSOCz8%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.******@***.***>>
